include/fbgemm/FbgemmBuild.h

```diff
@@ -26,7 +26,7 @@
       #endif
     #endif
   #else
-    #if __clang__ || __GNUC__ >=4 || __INTEL_COMPILER
+    #if __clang__ || __GNUC__ >=6 || __INTEL_COMPILER
```

Comment:
Please add a comment here why we need GNUC > 6.

Reply:
added